docs/examples/app-with-nomad/mycloneset-adopted.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: mycloneset-adopted
+  namespace: default
+spec:
+  components:
+  - name: mycloneset
+    properties:
+      apiVersion: apps.kruise.io/v1alpha1
+      kind: CloneSet
+      metadata:
+        name: mycloneset
+        namespace: default
+      spec:
```

Comment:
These boilerplate lines should be removed and they should be stored in component definition.